package.json

```diff
@@ -20,8 +20,8 @@
     "lint": "yarn lint:fix",
     "lint:fix": "yarn lint:typescript:fix && yarn lint:go:fix ",
     "lint:nofix": "yarn lint:typescript:nofix && yarn lint:go:nofix",
-    "lint:typescript:fix": "cross-env BROWSERSLIST_IGNORE_OLD_DATA=1 eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue --fix --report-unused-disable-directives .",
-    "lint:typescript:nofix": "cross-env BROWSERSLIST_IGNORE_OLD_DATA=1 eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue --report-unused-disable-directives --max-warnings=0 .",
+    "lint:typescript:fix": "cross-env BROWSERSLIST_IGNORE_OLD_DATA=1 NODE_OPTIONS=--max_old_space_size=4096 eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue --fix --report-unused-disable-directives .",
+    "lint:typescript:nofix": "cross-env BROWSERSLIST_IGNORE_OLD_DATA=1 NODE_OPTIONS=--max_old_space_size=4096 eslint --ignore-path=.gitignore --ext mjs,js,ts,tsx,vue --report-unused-disable-directives --max-warnings=0 .",
```

Comment:
Not sure how this is related to the current PR.

Why is this needed here, but its absence doesn't seem to cause any failures on `main`?

Reply:
We did not run `yarn lint` on macOS before, but we do now.  On the mac machines (only) it seems to have issues there, probably because those machines have less memory (and the default amount is probably derived from total memory).